src/screens/TransactionConfirmationScreen.js

```diff
@@ -207,7 +246,9 @@ const TransactionConfirmationScreen = () => {
         dispatch(updateTransactionCountNonce(maxTxnCount + 1));
         const txDetails = {
           amount: get(displayDetails, 'request.value'),
-          asset: get(displayDetails, 'request.asset'),
+          // Assume it's an ETH transaction
+          // Once it confirmed, zerion will show the data
+          asset: ethereumUtils.getAsset(allAssets),
```

Comment:
We might want to first check request.asset and if it doesn't exist, then use ETH. Or perhaps a better place for handling the missing asset case would be in the parsers/requests.js which constructs the "displayDetails" in the first place. If this were a token transfer, it would override the token data we were able to get and instead always show ETH. Around line 164 in parsers/requests.js, if we don't recognize it as an ETH or token transfer, and just see a generic "data" payload, then we could perhaps pass along the asset as ETH default there.